roles/bugziller/tasks/main.yml

```diff
@@ -0,0 +1,12 @@
+---
+- name: Install python-bugzilla
+  package: name=python-bugzilla state=latest
+
+- name: Setup python-bugzilla
+  command: "bugzilla login {{bugzilla_user}} {{bugzilla_pass}}"
+  args:
+    creates: "/home/jenkins/.bugzillacookies"
+  when: bugzilla_user is defined
+  no_log: True
+  become: yes
+  become_user: jenkins
```

Comment:
So we need to have the jenkins env being setup, so this should be a dependency, which in turn mean we can drop adding it to jenkins_nodes:children ?

Reply:
Yeah, we need a subset of the Jenkins environment setup. So we might as well do that.